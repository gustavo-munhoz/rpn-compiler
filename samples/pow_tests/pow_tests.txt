(2 1 ^)
(2 2 ^)
(2 3 ^)
(3 2 ^)
(1.5 2 ^)
(1.5 3 ^)
(10 3 ^)
(-2 1 ^)
(-2 2 ^)
(-2 3 ^)
(-3 2 ^)
(-1.5 2 ^)
(-1.5 3 ^)
(1 10 ^)
(-1 10 ^)
(-1 11 ^)
(0 5 ^)
(123 1 ^)
(-50 1 ^)
(3 11 ^)
(10 5 ^)
(2 15 ^)
(2 16 ^)
(-3 11 ^)
(-10 5 ^)
(-2 16 ^)
(0.5 14 ^)
(0.5 15 ^)
(0.5 100 ^)
(-0.5 14 ^)
(-0.5 15 ^)
(2 30 ^)
(1.1 30 ^)