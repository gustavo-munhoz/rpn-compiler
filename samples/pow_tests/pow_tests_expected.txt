0x4000
0x4400
0x4800
0x4880
0x4080
0x42D0
0x63D0
0xC000
0x4400
0xC800
0x4880
0x4080
0xC2D0
0x3C00
0x3C00
0xBC00
0x0000
0x57B0
0xD240
0x7C00
0x7C00
0x7800
0x7C00
0xFC00
0xFC00
0x7C00
0x0400
0x0000
0x0000
0x0400
0x8000
0x7C00
0x4C5C